docs/test-ui-bundle.md

```diff
@@ -0,0 +1,24 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with content from the ownCloud documentation, 
+build as usual the documentation with the antora command but add the option 
+`--ui-bundle-url <path-to-your-local-ui-bundle>/ui-bundle.zip` to your command.
+The location for this bundle is in the `docs-ui`reprository in directory `build/`.
```

Comment:
Did you fix the other whitespaces? Or was it a rendering issue? But here is still one missing.

Reply:
There was really one missing...